whylog/tests/tests_front/tests_whylog_factory.py

```diff
@@ -0,0 +1,27 @@
+import os
+import shutil
+from whylog.front.whylog_factory import whylog_factory
+from whylog.log_reader import LogReader
+from whylog.teacher import Teacher
+
+
+class MockClass():
```

Comment:
I can't see any use of this MockClass. It seems to be dead code. Remove it or add docstring about its purpose.